### ISSUE TITLE
MSB Mamm loan header report problem

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#1082]
Our MSB mammal loan invoice header has suddenly stopped working from the Edit Loan form - tries to download file as .swf.  Any idea what happened? I'd rather not have to recreate it in report builder, and I have pending loans held up by this.

**Comment by dustymc** [comment]
Something in your browser is not working properly.

**Comment by campmlc** [comment]
Yes, it appears to be something related to security upgrades with Firefox
and Chrome related to Shockwave Flave. I was able to get it to work in
Firefox by disabling all of the protections. Haven't figured out Chrome yet.
Could this be related to the http security issue? I am still getting
warnings when I go directly to arctos.database.museum.

**Comment by campmlc** [comment]
Jon and I still can't get our loan headers to download out of the Arctos loan forms in Chrome. The report tries to download as an swf file. I've updated Shockwave Flash and allowed plugin via Manage Exceptions. It does print after a convoluted series of steps, but then turns out to be some sort of video file? Here is screenshot. It is really frustrating to have to go to Firefox just to print out the loan headers, especially since Firefox thinks Arctos is unsecure. Has anyone else experienced this/figured out a way to solve?

![screenshot 2017-05-18 17 44 58](https://cloud.githubusercontent.com/assets/14808196/26227828/32568c6a-3bf2-11e7-8f17-1343ba2ff1dc.png)

**Comment by ekrimmel** [comment]
I can open your MSB loan headers when I am logged into Arctos in Chrome, Mariel, although they still open in Macromedia flash paper (in my browser). Have you double-checked that you are allowing flash in your [chrome settings](chrome://settings/content)? If you have the "block" box checked in the options below it might be preventing your Chrome from opening the loan header...? You could also add Arctos to the exceptions for blocking flash.

<img width="403" alt="screen shot 2017-05-19 at 5 57 52 am" src="https://cloud.githubusercontent.com/assets/11165319/26245302/1868d788-3c59-11e7-81ea-b25d9c87d8f5.png">

**Comment by jldunnum** [comment]
Hey Mariel,

Mine actually works now so it is an issue with individual settings.


______________________________________________________________

Jonathan L. Dunnum Ph.D.
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131
________________________________

**Comment by campmlc** [comment]
I can get it work but only by allowing Flash at all times.  I've added the
Arctos url copied as http://arctos.database.museum/ from the header as an
exception, and set it to ask first, and then it downloads it as an swf file
instead of opening.  Jon, what settings did you change?

**Comment by campmlc** [comment]
Dusty, you said there are versions of the forms that don't require
Shockwave Flash? Can we get these versions updated from the loan page
dropdown?

My computer crashed, and I have had to switch to an older one. I'm now
getting the same problem in Chrome with the MSB Bird loan header form, but
not the MSB Mamm form. It downloads as a swf file, with a warning, and
won't open.   I tried Firefox, but while it generates the form in the
browser it hangs up trying to print the pdf. I checked all the settings and
tried to add exceptions with no luck. It would be nice to have a
description of what to do in these cases, or get rid of Flash player
requirement for anything in Arctos.

**Comment by campmlc** [comment]
I'm having the same issue again with the MSB Bird loan invoice only; MSB Mamm works. To my knowledge nothing has been changed in these reports. I am trying to access from a computer that does not have Shockwave Flash. Can we get this form fixed so it does not required Flash? How do we do this? Help- I really don't want to have to start typing loan invoices by hand.

**Comment by dustymc** [comment]
The real solution is to move the documents to the CF Reporter where you can alter them like any other report.

I can change the hard-coded report type to PDF, and I've done so for MSBBirdLoanInvoice in test. If that helps I can do the same for MSBMammLoanInvoice and if not I can revert to flashpaper. Please try it out and let me know.
